scripts/release-bit-verification.sh

```diff
@@ -17,6 +17,9 @@ shout() {
 }
 # Check SHASUM for all the binary files and there should be no difference
 
+#For erroring out in case of error
+set -eo pipefail
```

Comment:
Can you move this as the first instruction (i.e., right after the shebang)? Just so it handles all subsequent commands below it.